docs/tutorial-studio.md

```diff
@@ -1,17 +1,17 @@
 ---
 id: tutorial-studio
-title: With Template Studio
+title: With Template Playground
 ---
 
-This tutorial will walk you through the steps of editing a clause template in [Template Studio](https://studio.accordproject.org/).
+This tutorial will walk you through the steps of editing a clause template in [Template Playground](https://playground.accordproject.org/).

```

Comment:
This tutorial can also be removed. We'll want to create a new one at a later date for the playground.